BoxOffice/MovieDTO/BoxOfficeResult.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  BoxOfficeResult.swift
+//  BoxOffice
+//
+//  Created by 이상윤 on 2023/04/24.
+//
+
+import Foundation
+
+// MARK: - BoxOfficeResult
+struct BoxOfficeResult: Decodable {
+    let boxofficeType, dateRange: String
```

Comment:
```suggestion
    let boxofficeType: String
    let dateRange: String
```

=> 변수가 서로 관련이 없는거라면 위처럼 따로 배치하는게 더 보기 좋을 것 같아요.

Reply:
넵! 적용하였습니다!

해당 커밋: 37c5ce1